spec/lib/search_spec.rb

```diff
@@ -139,33 +139,6 @@
     end
   end
 
-  context "with apostrophes" do
```

Comment:
Do we have another spec for searching text with apostrophes or should we keep one of these? (w/ modifications)

Reply:
We have other specs that test this case.